plugins/woocommerce/src/Internal/Admin/Settings.php

```diff
@@ -366,4 +369,15 @@ private function get_custom_settings( $settings ) {
 		}
 		return $settings;
 	}
+
+	/**
+	 * Check if current page is a analytics settings page
+	 * 
+	 * @return bool
+	 */
+	private function is_analytics_settings_page() {
```

Comment:
We can remove this function based on above comment in favor of `PageController::is_admin_page()`.